system-linux.c

```diff
@@ -2588,6 +2616,70 @@ system_if_dump_info(struct device *dev, struct blob_buf *b)
 	return 0;
 }
 
+int
+system_if_dump_info(struct device *dev, struct blob_buf *b)
+{
+	struct {
+		struct ethtool_link_settings req;
+		__u32 link_mode_data[3 * 127];
```

Comment:
Where is the magic value `3 * 127` coming from? Why `__u32` instead of `uint32_t`?